src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -1,67 +1,137 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
 @DisplayName("지하철역 관련 기능")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class StationAcceptanceTest {
+
+    @Autowired
+    private StationRepository stationRepository;
+
+    @BeforeEach
+    void setUp() {
+        stationRepository.truncateTableStation();
+    }
+
     /**
-     * When 지하철역을 생성하면
-     * Then 지하철역이 생성된다
-     * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
+     * When 지하철역을 생성하면 Then 지하철역이 생성된다 Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
      */
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
+        String stationName = "강남역";
 
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        createStationResponseBy(stationName);
 
         // then
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        List<String> stationNames = findStationsResponse()
+                .jsonPath()
+                .getList("name", String.class);
 
-        // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        assertThat(stationNames).containsAnyOf(stationName);
     }
 
-    /**
-     * Given 2개의 지하철역을 생성하고
-     * When 지하철역 목록을 조회하면
-     * Then 2개의 지하철역을 응답 받는다
-     */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다.")
+    @Test
+    void findStations() {
+        // given
+        String[] expectedStationNames = {"강남역", "양재역"};
 
-    /**
-     * Given 지하철역을 생성하고
-     * When 그 지하철역을 삭제하면
-     * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
-     */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+        createStationResponsesBy(expectedStationNames);
+
+        // when
+        ExtractableResponse<Response> response = findStationsResponse();
+
+        // Then
+        List<String> stationNames = response
+                .jsonPath()
+                .getList("name", String.class);
+
+        Assertions.assertAll(
+                () -> assertThat(stationNames.size()).isEqualTo(expectedStationNames.length),
+                () -> assertThat(stationNames).contains(expectedStationNames)
+        );
```

Comment:
다음과 같이 해볼 수도 있을 것 같아요 😄 
```suggestion
        assertThat(stationNames).hasSize(expectedStationNames.length)
                .contains(expectedStationNames);
```